examples/plot_nilearn_101.py

```diff
@@ -27,15 +30,14 @@
 anat_data = smooth_anat_img.get_data()
 print('anat_data has shape: %s' % str(anat_data.shape))
 anat_affine = smooth_anat_img.get_affine()
-print('anat_affineaffine:\n%s' % anat_affine)
-
-# Finally, it can be passed to nilearn function
-smooth_anat_img = image.smooth_img(smooth_anat_img, 3)
```

Comment:
> > -# Finally, it can be passed to nilearn function
> > -smooth_anat_img = image.smooth_img(smooth_anat_img, 3)

> I don't think we should remove this line as it shows that image.smooth_img
> function can take a nibabel object as input.

+1.

> I admit it's a bit weird to smooth an already smoothed object, though. Maybe
> the comment above could be more explicit or we could use img_index to retrieve
> a nibabel object which will then smoothed with a different factor (let's say
> 5). Finally both smoothed images could be compared in 2 plots.

I'd rather have a more explicit comment than making this more complex.